spacy/training/pretrain.py

```diff
@@ -31,6 +31,9 @@ def pretrain(
     allocator = config["training"]["gpu_allocator"]
     if use_gpu >= 0 and allocator:
         set_gpu_allocator(allocator)
+    if config["initialize"]["init_tok2vec"] is not None:
+        # TODO maybe print a warning?
```

Comment:
Maybe not a warning, but a note in the [API docs](https://spacy.io/api/data-formats#config-initialize) that this setting is ignored during pretraining itself?